### PR TITLE
ui: add some additional env level URLs for easier linking

### PR DESCRIPTION
**Pull request by johncowen**
Since we originally added these env variables we need to link to more varied URLs. These aren't used just yet be will be needed soonish.

### CODE REVIEW

File: ui-v2/config/environment.js [hunk 1]
```diff
@@ -64,6 +64,9 @@ module.exports = function(environment) {
       }
       return 'oss';
     })(),
+    CONSUL_HOME_URL: 'https://www.consul.io',
+    CONSUL_LEARN_URL: 'https://learn.hashicorp.com/consul',
+    CONSUL_API_URL: 'https://www.consul.io/api',
```
Comment (pearkes) [inline ui-v2/config/environment.js]:
```suggestion
    CONSUL_API_DOCUMENTATION_URL: 'https://www.consul.io/api',
```

Just given the potential here for `CONSUL_API` being seen or also the API address figured we should add that for clarity.

Comment (johncowen) [reply ui-v2/config/environment.js]: Ah yeah, good spot. You ok if I do it the other way round so `CONSUL_DOCUMENTATION_API_URL`. Actually, definitely shouldn't use `CONSUL_API_URL` - I think I want to change them to 

```
CONSUL_DOCUMENTATION_LEARN_URL
CONSUL_DOCUMENTATION_API_URL
CONSUL_DOCUMENTATION_URL
```

or 

```
CONSUL_DOCS_LEARN_URL
CONSUL_DOCS_API_URL
CONSUL_DOCS_URL
```

..might be a little easier on the fingers?

Comment (pearkes) [reply ui-v2/config/environment.js]: I like the shorter one 😄

Comment (johncowen) [reply ui-v2/config/environment.js]: Hey @pearkes I changed that now, thats way nicer and clearer, thanks for that!

If you can take one more look and gimme a quick thumbs up and I'll merge this.

**Pull request merged**